__init__.py

```diff
@@ -932,6 +930,9 @@ def unregister():
     from bpy.types import WindowManager
     global thangs_login
 
+    amplitude.send_amplitude_event(
+        "Thangs Blender Addon - Uninstalled", event_properties={})

```

Comment:
This won't do what we think, because `unregister` gets invoked just when closing blender.  I've been trying to find a way to detect when it's actually uninstalled, but haven't found anything yet.

Reply:
Really? I was uninstalling the addon and had a print statement that ran every time I tested.